bot.py

```diff
@@ -19,23 +19,24 @@
 
 
 
-bot = commands.Bot(command_prefix=bot_prefix, case_insensitive=True)
-
-
+bot = commands.Bot(
+    command_prefix=bot_prefix, 
+    case_insensitive=True, 
+    activity=discord.Activity(
+        type=discord.ActivityType.watching, 
+        name=f"{bot.command_prefix}help"
+    )
+bot.load_extension("cogs.music")
```

Comment:
Python3 will report this function as syntactically invalid because you are missing a closing parenthesis . Also, because bot is not currently defined, you will need to use instead ```name=f"{bot_prefix}help"``` in order to get the prefix.

Reply:
aight